src/BetModal.jsx

```diff
@@ -52,12 +52,12 @@ export default function BetModal({ open, onConfirmed }) {
         </div>
 
         <div style={actionsRowStyle}>
-          <button style={primaryBtnStyle(disabled)} onClick={place} disabled={disabled}>
-            Place Bet
-          </button>
           <button style={secondaryBtnStyle} onClick={() => navigate('/seat-buy-in')}>
             Reset Bank
           </button>
+          <button style={primaryBtnStyle(disabled)} onClick={place} disabled={disabled}>
```

Comment:
Could you go ahead and convert this to a tailwind class? I think that would be the best way to address the style objects going forward. I’m just still trying to learn tailwind so it’s harder for me to compose sophisticated classes so I end up using style objects a lot. I don’t want that to become a norm though. We should def be using the styling library we have configured. What do you think?

Reply:
I don't really see the benefit of splitting between style objects and tailwind classes. It's already confusing the hell out of me lol the only reason I haven't really added many tailwind classes is because I'm still learning it. Ideally we wouldn't have a split approach.